eventdata/challenges/daily-log-volume-index.json

```diff
@@ -30,12 +30,23 @@
         }
       }
     },
+    {
+      "name": "check-cluster-health",
+      "operation": {
```

Comment:
we should add a `"retry-until-success": true` here (similar to https://github.com/elastic/rally-eventdata-track/pull/97).